framework/include/cppmicroservices/Constants.h

```diff
@@ -21,11 +21,11 @@
 =============================================================================*/
 
 #ifndef CPPMICROSERVICES_CONSTANTS_H
-#define CPPMICROSERVICES_CONSTANTS_H
+#  define CPPMICROSERVICES_CONSTANTS_H
 
-#include "cppmicroservices/FrameworkConfig.h"
+#  include "cppmicroservices/FrameworkConfig.h"
```

Comment:
Is this the result of running clang-format ?? This style looks wierd. a space after #

Reply:
Done.